services/ec2/src/main/java/org/finra/gatekeeper/services/aws/SnsService.java

```diff
@@ -42,11 +44,14 @@
 
     private final AwsSessionService awsSessionService;
     private final GatekeeperSnsProperties gatekeeperSnsProperties;
+    private final GatekeeperEC2Properties gatekeeperEC2Properties;
 
     @Autowired
     public SnsService(AwsSessionService awsSessionService,
+                      GatekeeperEC2Properties gatekeeperEC2Properties,
```

Comment:
I think what is stored in gatekeeperEC2Properties should just be stored in gatekeeperSnsProperties